site/content/user-guide/amqp-adapter.md

```diff
@@ -275,6 +275,35 @@ The table below provides an overview of the properties that must be set on a com
 | *correlation-id* | yes             | *properties*             | *string*     | MUST contain the value of the *correlation-id* property of the command request message. |
 | *status*         | yes             | *application-properties* | *integer*    | MUST contain a status code indicating the outcome of processing the command at the device (see [Command & Control API]({{< ref "/api/Command-And-Control-API.md" >}}) for details). |
 
+### Examples
+
+The AMQP Command-line client can be used to receive and send a command response message from/to a consumer application. The client is used to simulate a device to receive commands and for sending command responses back to the application. The command consumer client is used to simulate an application sending a command to the device and receiving a command response from the device.
+
+Start the AMQP Command-line client, as follows:
+
+    $ /hono/cli/target$ java -jar hono-cli-0.9-M1-SNAPSHOT-exec.jar --spring.profiles.active=amqp-adapter-cli --subscribe=true
+    $ 04:13:49.651 [vert.x-eventloop-thread-0] INFO  org.eclipse.hono.cli.AmqpSend - Device is now ready to receive commands
+
+After successfully starting the client, a message indicating that the device is ready to receive commands will be printed to standard output. The device is now waiting to receive commands from applications. To send a command to the device, open a new terminal shell and start the command consumer, as shown below:
+
+    $ /hono/cli/target$ mvn spring-boot:run -Drun.arguments=--hono.client.host=localhost,--hono.client.username=consumer@HONO,--hono.client.password=verysecret -Drun.profiles=command,ssl
```

Comment:
FMPOV it is very confusing that we start the client using `java -jar` in the first step and then start the client using `mvn spring-boot:run` in the second step. I would actually favor using `java -jar` everywhere ...

Reply:
you mean starting the command consumer using `java -jar`? using the `mvn spring-boot:run` approach, for starting the command consumer, is how it is documented on the hono website

Reply:
both ways work, no doubt. I just propose to start the consumer consistently using the same approach throughout the example and my personal preference would be to use the `java -jar` form because it is more compact and intuitive.

Reply:
ok 👍